### PR TITLE
Fixes #110 Enable pipelining of Bash Script output to UI

### LINKED ISSUE #110: Enable pipelining of Bash script output to UI
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

The UI receives a response after the script is executed. This creates trouble deploying the app to Heroku. 
### Expected behaviour

Receive responses  from the the server, executing the script command by command

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

Issue #110 
## Motivation and Context

This change fixes the issue with inability of deployment to heroku due to Request Timeout.
Also, it gives a visual description to the user of what is happening BTS.

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by imujjwal96** [comment]
@pri22296 @sch00lb0y

**Comment by pri22296** [comment]
@imujjwal96 Please resolve merge conflicts

**Comment by imujjwal96** [comment]
@pri22296 Resolved

File: WebUI/generate.sh [hunk 1]
```diff
@@ -50,19 +48,23 @@ cd ${BUILD_DIR}
 mkdir _themes
 
 # Install packages required for documentation generation
+echo -e "Installing packages required for documentation generation\n"
 pip install -q -r requirements.txt
+echo -e "Generation successful\n"
```
Comment (pri22296) [inline WebUI/generate.sh]: shouldn't this be `Installation successful`. other than that Looks good.

Comment (imujjwal96) [inline WebUI/generate.sh]: Updated

**Comment by sch00lb0y** [comment]
@imujjwal96 could you squash 3 commit into single commit.

**Comment by imujjwal96** [comment]
@sch00lb0y if you'll approve the changes, i'll squash and merge from github

**Comment by sch00lb0y** [comment]
@imujjwal96 don't merge now. maintainer will do

**Comment by niranjan94** [comment]
> @sch00lb0y if you'll approve the changes, i'll squash and merge from github 

Don't depend on GHs squash and merge option. It's your responsibility to squash commits in PR _before_ merge.

**Comment by imujjwal96** [comment]
I have trouble squashing the commits when I Resolve merge conflicts. Can you please tell me how should I squash them? or maybe any tutorial for it.
@niranjan94

**Comment by niranjan94** [comment]
- The best way is `git rebase -i <hash-of-the-commit-to-start-from>` and just marking the commits you wanna squash/pick, saving it and force pushing.
- Another way would be to do a soft reset until before the commits you wanna squash. Then create a new commit with all the changes and force push. (this only applies when the commits you wanna squash are in-line from top)

**Comment by imujjwal96** [comment]
I tried the first method and this happened

image

**Pull request merged**